src/dispatch.rs

```diff
@@ -169,10 +171,12 @@ impl<'t> Dispatcher<'t> {
 
         #[cfg(target_os = "emscripten")]
         self.dispatch_seq(res);
+
+        self.dispatch_thread_local(res);
```

Comment:
Actually it's not the case for `AsyncDispatcher` since you can't execute thread local systems while executing asynchronously, so i might just leave `AsyncDispatcher` as it was.